Patch to ensure SSL will be picked up by EasyBuild and not the OS. Important for older OpenSSL versions
Author: J. Sassmannshausen / GSTT
diff --git a/userApps.orig/kent/src/inc/common.mk b/userApps/kent/src/inc/common.mk
index d200b43..cb257d6 100644
--- a/userApps.orig/kent/src/inc/common.mk
+++ b/userApps/kent/src/inc/common.mk
@@ -133,7 +133,7 @@ endif
 ifneq (${SSL_DIR}, "/usr/include/openssl")
   ifneq ($(UNAME_S),Darwin)
     ifneq ($(wildcard ${SSL_DIR}),)
-      L+=-L${SSL_DIR}/lib
+      L+=-L${SSL_DIR}/lib -lssl -lcrypto
     endif
   endif
     HG_INC+=-I${SSL_DIR}/include
@@ -142,29 +142,6 @@ endif
 ifeq (${IS_HGWDEV},yes)
    L+=/hive/groups/browser/freetype/freetype-2.10.0/objs/.libs/libfreetype.a -lbz2
    L+=/usr/lib64/libssl.a /usr/lib64/libcrypto.a -lkrb5 -lk5crypto -ldl
-else
-   ifneq ($(wildcard /opt/local/lib/libssl.a),)
-       L+=/opt/local/lib/libssl.a
-   else
-     ifneq ($(wildcard /usr/lib/x86_64-linux-gnu/libssl.a),)
-	L+=/usr/lib/x86_64-linux-gnu/libssl.a
-     else
-        ifneq ($(wildcard /usr/local/opt/openssl/lib/libssl.a),)
-           L+=/usr/local/opt/openssl/lib/libssl.a
-        else
-           L+=-lssl
-        endif
-     endif
-   endif
-   ifneq ($(wildcard /opt/local/lib/libcrypto.a),)
-       L+=/opt/local/lib/libcrypto.a
-   else
-        ifneq ($(wildcard /usr/local/opt/openssl/lib/libcrypto.a),)
-           L+=/usr/local/opt/openssl/lib/libcrypto.a
-        else
-           L+=-lcrypto
-        endif
-   endif
 endif
 
 # autodetect where libm is installed
